blog/2020-04-02-march-dev-update.md

```diff
@@ -0,0 +1,46 @@
+---
+title:  "March, 2020 - Dev Update"
+author: daniel
+author_url: https://github.com/da-kami
+author_image_url: https://avatars1.githubusercontent.com/u/5557790
+tags: [dev-update]
+---
+
+> The sky is everywhere, it begins at your feet.
+>
+> — Jandy Nelson
+
+[Join us for the next COMIT Community ☕️](https://meet.jit.si/COMIT-Community-Coffee-04-2020) today, 7pm Sydney time, or listen in on [youtube](https://youtu.be/DHbGsf9i8TM)!
+
+Today's community coffee we will reflect on the first quarter of this year with a focus on Lightning Network (layer 2) integration into COMIT.
+We will discuss advantages and possible pitfalls. 
```

Comment:
:question: Advantages of what?
:thinking: Rather than "possible pitfalls" I would maybe frame it as "technical challenges we encountered", unless you're referring to something else and I'm missing the point.